react-native/.circleci/config.yml

```diff
@@ -0,0 +1,351 @@
+version: 2.1
```

Comment:
also would be good if you create some diagram-picture with steps flow to visualize it.